### PR TITLE
TT-1768: First draft of documentation

### PR DESCRIPTION
**Pull request by J-Lambo**
Documented database, matching strategy, and basic run instructions for example local matching service. 

With input from: @SKeerthana @andy-paine

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,137 @@
...
+
+If the example finds a match in Cycles 1 or 3 it will store the PID in the database.
+
+**Caveats**
+
+This example local matching service will only return `match` or `no match`. It does not provide any fuzzy matching, for example matching `ONeill` to `O'Neill` or `Will` to `William`. The example does not provide any means of logging or investigating results.
+
+The example ignores any attributes where `verified` is set to `false`. This would be a suitable starting point for any service with an existing dataset expected to include most new users.
+
+The example will ignore any verified historical
```
Comment (SKeerthana) [inline README.md]: It only ignores any unverified attribute as mentioned above. We are not ignoring the "verified" historical attributes.

**Comment by J-Lambo** [comment]
Thanks @SKeerthana. Good spot! I've amended

File: README.md [hunk 2]
```diff
@@ -1,4 +1,145 @@
-# Verify Local Matching Service Example
+# Example local matching service for GOV.UK Verify
 
-This is an example of local matching service that could be used by relaying parties as a reference when building their matching service.
+This is an example local matching service built by the GOV.UK Verify team to demonstrate how a local matching service works and help services building their own. Every government service connecting to Verify must build their own local matching service.
```
Comment (andy-paine) [inline README.md]: I feel like `must build their own` sounds a bit anti-collaborative. Maybe just `must build a local matching service` as in an ideal world we would actually have us and RPs sharing code/helping each other 😄

File: README.md [hunk 3]
```diff
@@ -1,4 +1,145 @@
...
+
+This example uses a simple database and matching strategy. It has not been designed to be used in front of a real database.
+
+See the [examples folder](/examples/) for example matching datasets.
+
+### Database
+
+This example local matching service uses a SQL database. Refer to the [database schema](/docs/schema.png) for details.
+
+All SQL has been written for Postgres and has not been tested with any engine.
```
Comment (andy-paine) [inline README.md]: Should say any other engine - we've tested it in Postgres at least

**Pull request merged**